zonemaster/zonemaster-backend
# Update MANIFEST files

## Purpose

This PR does some cleaning up in the MANIFEST files.

## Changes

Added to the dist tarball:
* newly added unit tests.

Removed from the dist tarball:
* the Travis yaml file, and
* newly deleted files.

New files that aren't added to the dist tarball:
* newly added Github related stuff,
* newly added internal documentation.

Other changes:
* the regex `maint` is replaced with `^maint/`, and
* the regex `.last_cover_stats` is replaced with `^\.last_cover_stats$`.

## How to test this PR

Make sure distcheck doesn't report any problems:
```sh
git clean -dfx
perl Makefile.PL
make distcheck
```
